shim_et/xplat/executorch/kernels/portable/op_registration_util.bzl

```diff
@@ -1330,4 +1337,4 @@ def portable_source_list():
 
 def portable_header_list():
     """All the header file names from //executorch/kernels/portable/cpu/"""
-    return ["selective_build.h", "scalar_utils.h", "math_constants.h", "vec_ops.h"]
+    return ["selective_build.h", "scalar_utils.h", "math_constants.h", "vec_ops.h"]
```

Comment:
not 100% sure, but it looks like github is saying you removed the end-of-file newline? if so, please put it back